main.qml

```diff
@@ -462,7 +515,8 @@ ApplicationWindow {
     function onWalletMoneyReceived(txId, amount) {
         // refresh transaction history here
         currentWallet.refresh()
-        currentWallet.history.refresh() // this will refresh model
+        console.log("Confirmed money found")
```

Comment:
Why not use the existing logs ? Or is this somehow plugged into those ?

Reply:
yes. This is the Qt way of doing it. It's also forwarded to the wallet2 log.